change/@fluentui-react-radio-c738372a-484c-42ab-b64c-20287e3f4373.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "write readme",
```

Comment:
_nit:_
```suggestion
  "comment": "Adding readme.",
```

Reply:
I went back n forth on this but _technically_ it was already added and just written :)